app/models/host/managed_extensions.rb

```diff
@@ -17,7 +17,10 @@ def queue_reboot
   end
 
   def setReboot
-    old.becomes(Host::Discovered).reboot
+    old.becomes(Host::Discovered).reboot if errors.empty?
```

Comment:
i dont get it.
if errors are empty, orchestration should not trigger... which errors do you have ? is it simple validation? something else?